test/cssModules.transform.js

```diff
@@ -13,7 +14,7 @@ module.exports = {
 
     const cssModule = {};
     classNames.forEach(className => {
-      cssModule[className] = className;
+      cssModule[className] = `${fileName}__${className}`;
```

Comment:
Just to make the review process a little easier, this file is the core of the change. The rest of the PR is just updated test code and snapshots to accomodate it.